common/scripts/docker-build.sh

```diff
@@ -188,7 +188,8 @@ popd > /dev/null 2>&1
 
 # validate docker version against minimum required docker version
 # docker_version=$(docker version --format '{{.Server.Version}}')
-docker_version=$(docker version | grep 'Version:' | awk '{print $2}')
+docker_version=$(docker version)
+docker_version=$(echo "$docker_version" | grep 'Version:' | awk '{print $2}')
```

Comment:
@chamilad @ravihansa3000 I completely missed the earlier commits, I am going to re-fork, commit this work and then issue a new PR. Please ignore the current PR.